app/scripts/controllers/projects.js

```diff
@@ -111,7 +111,12 @@ angular.module('openshiftConsole')
 
     $scope.newProjectPanelShown = false;
 
-    $scope.createProject = function() {
+    $scope.createProject = function(event) {
+      var button =_.get(event, 'target');
+      while (button && !$(button).hasClass('btn')) {
```

Comment:
It is searching for the button as a parent of the clicked element.

I will change to use angular.element rather than jquery.